examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -74,75 +97,104 @@ my_module/my_module.py       4      0   100%
 TOTAL                        5      0   100%
 ```
 
-Now that your tests are working locally, you can configure Cloud Builder to run them every time you push new code.
+Now that your tests are working locally, you can configure Google Cloud Build to run them every time you push new code.
 
 ## 3. Building a Python Cloud Build Container
-To run Python based tests in cloud builder you need a Python [cloud builder](https://cloud.google.com/cloud-build/docs/cloud-builders). 
-A cloud builder is just a docker container image with the software you need in it. Google doesn't distribute a 
-Python cloud build container, so you have to build one yourself. The code needed to build a custom Python 3 cloud build 
+To run Python-based tests in Google Cloud Build you need a Python container image used as a [cloud builder](https://cloud.google.com/cloud-build/docs/cloud-builders). 
+A cloud builder is just a container image with the software you need for your build steps in it. Google does not distribute a 
+prebuilt Python cloud builder, so a custom cloud builder is required. The code needed to build a custom Python 3 cloud build 
 container is located in '/python-cloud-builder' under the root of this project. 
 
-Run these commands from the root of the project to build and upload your custom Python cloud build container:
+Inside that folder you will find a Dockerfile and a very minimal cloud-build.yaml.
 
+The Dockerfile specifies the software that will be inside the container image.
+
+```Docker
+FROM python:3 # Start from the public Python 3 image contained in DockerHub
+RUN pip install virtualenv 
+# install virtualenv so that a virtual environment can be used to carry build steps forward.  
```

Comment:
Capitalize "install"

Reply:
Done.  Fixed a few more of these as well.